pkg/serverless/daemon/routes.go

```diff
@@ -71,10 +71,17 @@ func (e *EndInvocation) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	log.Debug("Hit on the serverless.EndInvocation route.")
 	endTime := time.Now()
 	ecs := e.daemon.ExecutionContext.GetCurrentState()
+	responseBody, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Error("Could not read EndInvocation request body")
+		http.Error(w, "Could not read EndInvocation request body", 400)
```

Comment:
Log.Error return the error after printing it.
```suggestion
		err := log.Error("Could not read EndInvocation request body")
		http.Error(w, err.Error(), 400)
```